app/javascript/stimulus/controllers/article_controller.js

```diff
@@ -0,0 +1,30 @@
+import { Controller } from 'stimulus';
```

Comment:
look how beautiful this code is, I'm almost crying! 😭

ps. would it be a good time to add method comments? by looking it at without context i have no idea what `boost` or `sink` do :D

Refactorings in general are a great moment to think about these things

Reply:
Good idea, I believe I meant to change these method names from the functions declared in the old JS, but forgot. :slightly_smiling_face: I will definitely do that!